makerspace/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from web.multilingual.database import MultiLingualTextField, MultiLingualRichTextUploadingField
+
+
+class Tool(models.Model):
+    title = MultiLingualTextField(
+        max_length=100,
+        verbose_name=('Title'),
+    )
+    image = models.ImageField(verbose_name=('Image'), blank=True, )
+    content = MultiLingualRichTextUploadingField()
+
+
+class Makerspace(models.Model):
```

Comment:
Comments have been added

Reply:
What do you mean by add description?

Reply:
The intention of having two different classes is to distinguish the usage.  Makerspace model is used to MakerspaceView, while tool model is used to the tool view.  

Makerspace - A view of  description of makerverkstedet and href to tools
Tools -  A view of all available tools.

Reply:
I understand the benefits of distinguishing the different usages. However, the Makerspace model will only ever be used for one single object, which is what made me initially wonder about why the model exists. For what it is worth, I think that since the image is not used, the model is for all intents and purposes equivalent to the ContentBox model. This model is designed for purposes like this, and I would say that using it instead would be better.